ci/sanity/install.sh

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env bash
+
```

Comment:
though not related to current PR, whenever we are touching any existing files, if a copyright message is missing, let us add one.